ground/src/main/java/com/google/android/ground/persistence/local/room/converter/LoiPropertiesMapConverter.kt

```diff
@@ -0,0 +1,45 @@
+package com.google.android.ground.persistence.local.room.converter
+
+import androidx.room.TypeConverter
+import com.google.common.reflect.TypeToken
+import com.google.gson.Gson
+import com.google.gson.JsonSyntaxException
+import kotlinx.collections.immutable.toPersistentMap
+import org.json.JSONException
+import org.json.JSONObject
+import timber.log.Timber
+
+
+object LoiPropertiesMapConverter {
+  @TypeConverter
+  fun toString(properties: Map<String, Any>?): String? {
+    if (properties ==null) {
```

Comment:
Nit: Run gradle ktfmtFormat (also avail in Gradle pane in Android Studio) to autoformat.

Reply:
To avoid needing to handle both empty and null states, can we make these non-null and default to empty?

Reply:
Done